libopenage/util/fixed_point.h

```diff
@@ -446,8 +447,37 @@ class FixedPoint {
 		return is;
 	}
 
+	/**
+	 * Pure FixedPoint sqrt implementation using Heron's Algorithm.
+	 *
+	 * Note that this function is undefined for negative values.
```

Comment:
Would also be nice if you add the other limitation from your PR description to the docstring:

> There's a small loss in precision depending on the value of fractional_bits and the position of the most significant bit: if the integer portion is very large, we won't have as much (absolute) precision. Ideally you would want the intermediate_type to be twice the size of raw_type to avoid any losses.